src/Workspaces/Core/Portable/EmbeddedLanguages/RegularExpressions/LanguageServices/RegexEmbeddedLanguage.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.EmbeddedLanguages.LanguageServices;
+using Microsoft.CodeAnalysis.EmbeddedLanguages.VirtualChars;
+using Microsoft.CodeAnalysis.LanguageServices;
+
+namespace Microsoft.CodeAnalysis.EmbeddedLanguages.RegularExpressions.LanguageServices
+{
+    internal class RegexEmbeddedLanguage : IEmbeddedLanguage
+    {
+        public int StringLiteralKind { get; }
+        public ISyntaxFactsService SyntaxFacts { get; }
```

Comment:
>SyntaxFacts [](start = 35, length = 11)

Can you verify all these properties are used? I suspect a number of them are not.

Reply:
Have verified they are all used.